Fixed yuiseki's svg renderer

DerAndereJohannes opened the pull request:
### Overview
- Completed the svg renderer to work properly.
- Created a simple unit test file to check validity.

### Potential Improvements
- size scaling of pieces.
- improve numbering text for pieces on hand.
- not all pieces are perfectly central.

### Example
```python
import shogi
b = shogi.Board('4+R3l/1r1+P2gk1/3p1p1s1/2pg3pp/1p4p2/SP4PPP/2NP1PKS1/2G2+n3/8L w B2N2L3Pbgsp 10')
with open('svg_test.svg', 'w') as tfile:
	print(b.svg(), file=tfile)
```
![complex_board](https://user-images.githubusercontent.com/25019880/108822633-a3aa9180-75bf-11eb-96b5-168baf83d92e.png)


Any comments welcome!

gunyarakun commented:
@DerAndereJohannes 
Thank you for making your pull request based on the work of @yuiseki .

gunyarakun commented on README.rst:
```diff
@@ -105,6 +105,15 @@ Features
+  .. code:: python
+
+      >>> print(board.svg())
+      <svg>
```
It would be great if we could embed a real image generated by the command `board.svg()`.
But I know it's not easy.

gunyarakun reviewed (approved):
LGTM!

gunyarakun commented:
@yuiseki I've heard the license of the koma SVGs are `Creative Commons BY-SA License`. Where is the source url you fetched the SVGs?

yuiseki commented:
@gunyarakun I've copied those koma SVGs from this repository and merged it at SVG.py as definition of SGV.
https://github.com/orangain/shogi-piece-images/tree/master/dist

It seems originally work is one of contribute for wikipedia.
https://commons.wikimedia.org/wiki/File:Shogi_kinsho(svg).svg

I've already contain those license information to SVG.py

CC BY-SA 4.0 is compatible with GPLv3, but those SVG is licensed under CC BY-SA 3.0, So license matter isn't be simple...
https://creativecommons.org/2015/10/08/cc-by-sa-4-0-now-one-way-compatible-with-gplv3/

yuiseki commented:
I'm thinking one of solution to this license problem: Separate shogi.svg file as independent file and license only it as CC BY-SA 3.0 then read it from SVG.py

DerAndereJohannes commented:
I added the svg image and some more test code into the README! (the :align: center did not really work though..) I found another bug too when a player has more than 9 of a piece in their hand and fixed it! 

Should we proceed with the yuiseki's idea of keeping the svg file separate with the other lisence? Is this legally possible? If it is then I think it is also the correct solution

gunyarakun commented:
I forgot to write my thought.
I guess the best way is not putting SVG files in this repository but write a instruction to fetch SVG files in README.md and user can pass the place of SVG files through an argument of `svg()`.
